crates/core/Cargo.toml

```diff
@@ -63,8 +63,9 @@ tokio = { workspace = true }
 # datafusion
 datafusion = { workspace = true, optional = true }
 datafusion-expr = { workspace = true, optional = true }
-datafusion-common = { workspace = true, optional = true }
+datafusion-common = { workspace = true }
 datafusion-physical-expr = { workspace = true, optional = true }
+log = "0.4.22"
```

Comment:
can this dep be added to workspace for centralized management?

Reply:
oh, forgot it, will update it.